### PR TITLE
Add updated OAH tools article

### PR DESCRIPTION
**Pull request by willbarton**
[The official announcement is out.](http://www.consumerfinance.gov/blog/know-before-you-owe-making-the-mortgage-process-easier-for-you/)

This PR adds an article by Catherine Farman, Amy Mok, and Stephanie Osan about the updated OAH tools launching today. It's been pre-cleared by SMEs, Legal, Regs, Audrey, Ashwin, and the editorial board.

You can view it [in my fork](http://willbarton.github.io/cfpb.github.io/articles/updated-oah-tool-guides-users-through-homebuying-process).

### CODE REVIEW

File: _posts/2015-09-17-updated-oah-tool-guides-users-through-homebuying-process.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+published: true
+layout: post
+author: Catherine Farman, Amy Mok, Stephanie Osan
```
Comment (KimberlyMunoz) [inline _posts/2015-09-17-updated-oah-tool-guides-users-through-homebuying-process.md]: Include an `and` between Amy and Stephanie.

**Comment by stephanieosan** [comment]
Could we add titles to the screenshots? Maybe these, and they could be in a different style type than the regular paragraph copy so they stand out a little (h4 or h5, or even just bolded).

"First iteration of a phase page layout"

"Final design for a phase page"

"Additional content in the action steps"

**Comment by willbarton** [comment]
@stephanieosan Absolutely. Let me see what I can do.

**Comment by willbarton** [comment]
Alright, I added the titles. They are in italics under the images, similar to [the article on pre-paid disclosures](http://cfpb.github.io/articles/designing-prepaid-card-disclosures/).

**Comment by ascott1** [comment]
:house_with_garden: :+1: :tada: 

:cat2:

**Pull request merged**